### PR TITLE
KYLIN-5225 fix wrong name and classpath in shell in kylin5.0

### PR DESCRIPTION
**Pull request by zznlime**
## Proposed changes

Fix some wrong classpath in .sh file at build/bin of kylin5.0.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] I have create an issue on [Kylin's jira](https://issues.apache.org/jira/browse/KYLIN-5226), and have described the bug there in detail

### CODE REVIEW

**Comment by zznlime** [comment]
@xiaowangyu

**Comment by hit-lacus** [comment]
Thanks for contributions

File: build/bin/upgrade.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@
 function help() {
```
Comment (Mukvin) [inline build/bin/upgrade.sh:20]: This file can be removed, it is useless now.

Comment (zznlime) [reply build/bin/upgrade.sh]: Done. upgrade.sh has been removed.

**Comment by Mukvin** [comment]
LGTM

**Pull request merged**